### PR TITLE
Codestyle

### PR DESCRIPTION
**Pull request by wilsonge**
More codestyle fixes. There is a failure in the com_postinstall model due to all the extra info that @nikosdion added

Anyone got any ideas? Tbh I think it's more the sniff - but anything good ideas appreciated

### CODE REVIEW

**Comment by mbabker** [comment]
I sent you a patch for that model.  Converts all the alignment stuff to spaces and makes liberal use of our 150 character line length to move that `@see` tag the checker is griping about so that it isn't the first content on a line which throws the rest of that doc block's checks out of whack.

**Comment by wilsonge** [comment]
Looks good to me :) ty!

File: build/phpcs/Joomla/ruleset.xml [hunk 1]
```diff
@@ -113,7 +113,6 @@
 
  <rule ref="Joomla.Commenting.FunctionComment">
   <!-- We only want this for libraries, language and cli for now -->
-  <exclude-pattern type="relative">components/*</exclude-pattern>
```
Comment (Kubik-Rubik) [inline build/phpcs/Joomla/ruleset.xml]: @wilsonge If we remove this line, shouldn't the comment above also be updated?

Comment (wilsonge) [inline build/phpcs/Joomla/ruleset.xml]: Just merge it and I'll update the comments in the file in a separate PR - there's a bunch of places where we've removed exceptions and not updated the comments associated lol

**Comment by Kubik-Rubik** [comment]
Thank you @wilsonge. Merged with https://github.com/joomla/joomla-cms/commit/8384a5886bba57336b1c4357be63202837185574

**Pull request closed without merging**